packages/vite-plugin-cloudflare/src/miniflare-options.ts

```diff
@@ -303,14 +303,14 @@ export function getDevMiniflareOptions(
 
 									// Sometimes Vite fails to resolve built-ins and converts them to "url-friendly" ids
 									// that start with `/@id/...`.
-									if (moduleId.startsWith('/@id/')) {
-										const result = {
-											externalize: moduleId.slice('/@id/'.length),
-											type: 'builtin',
-										} satisfies vite.FetchResult;
-
-										return new MiniflareResponse(JSON.stringify({ result }));
-									}
+									// if (moduleId.startsWith('/@id/')) {
```

Comment:
Can you add a comment here explaining why this was here and why it has been commented out?

Reply:
c16f92f